Tools/MiniBrowser/win/Common.cpp

```diff
@@ -78,6 +78,17 @@ bool getAppDataFolder(_bstr_t& directory)
     return true;
 }
 
+bool getKnownFolderPath(REFKNOWNFOLDERID id, std::wstring& knownFolderPath)
+{
+    PWSTR path;
+
+    if (FAILED(SHGetKnownFolderPath(id, KF_FLAG_CREATE, 0, &path)))
```

Comment:
You need to free `path` with `CoTaskMemFree`. See https://learn.microsoft.com/en-us/windows/win32/api/shlobj_core/nf-shlobj_core-shgetknownfolderpath

Reply:
Good catch! Fixed.